content/en/core/overview/data-flows-for-analytics/cht-sync.md

```diff
@@ -31,7 +31,7 @@ A free and open source SQL database used for analytics queries. See more at the
 
 #### Superset
 
-[Apache Superset](https://superset.apache.org/) is a free an open source platform for creating data dashboards.
+We recommend [Apache Superset](https://superset.apache.org/) as the Data Visualization Tool, it is a free an open source platform for creating data dashboards.
```

Comment:
@Phil-Mwago what was omitted , was it not Superset. Kindly clarify.

Reply:
@esthermmoturi yes, superset will not be part of the bundled containers in the cht-sync config, however we will recommend it as the visualization tool since its open source and a provide a script to connect.